src/schema/partner_show.ts

```diff
@@ -25,15 +30,33 @@ import {
 } from "graphql"
 import { allViaLoader } from "../lib/all"
 import { totalViaLoader } from "lib/total"
+import { connectionFromArraySlice } from "graphql-relay"
 
 const kind = ({ artists, fair }) => {
   if (isExisty(fair)) return "fair"
   if (artists.length > 1) return "group"
   if (artists.length === 1) return "solo"
 }
 
+const artworksArgs = {
+  exclude: {
+    type: new GraphQLList(GraphQLString),
+    description:
+      "List of artwork IDs to exclude from the response (irrespective of size)",
+  },
+  for_sale: {
+    type: GraphQLBoolean,
+    default: false,
+  },
+  published: {
+    type: GraphQLBoolean,
+    defaultValue: true,
+  },
+}
+
 const PartnerShowType = new GraphQLObjectType({
   name: "PartnerShow",
+  // @ts-ignore
   deprecationReason: "Prefer to use Show schema",
```

Comment:
Actually, `Show` is the better/more modern type to use. There was some breaking schema changes at some point that necessitated `PartnerShow` forking to `Show`.

So maybe can just nix these and use `show`? (sorry for not noticing before this was the file being used)

Reply:
Cool to keep these updates just in case? I've revised `show` to the modern connection form but this (could) be of use at some later point?

Reply:
(Can also delete to just reduce number of changes)